### PR TITLE
Feature/73336

### PR DESCRIPTION
**Pull request by anqaka**
intenal dev-73336

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -481,3 +481,73 @@ If responisve image aspect ratio is added, additional styles inline are generate
...
+Magepack config will be generated in `/tools` as `magepack.config.js` (which is a symlink to vendor/snowdog/frontools/magepack.config.js).
+You can move this file to main repo or to other location, add this to `.gitignore` and commit changes.
+
+With commited magepack config, during deployment, after assets compilation, run magepack bundling:
+```
+yarn magepackBundle --config <config_path>
+```
+
+### Add assets
+* If you added fonts or external assest that can be load with `preload`, add them in:
```
Comment (JKrupinski) [inline README.md:509]: There is a typo `assest`, should be `assets`

File: README.md [hunk 2]
```diff
@@ -481,3 +481,73 @@ If responisve image aspect ratio is added, additional styles inline are generate
...
+    as="style"
+  />
+  ```
+  an example can be found here: `vendor/snowdog/theme-frontend-alpaca/Amasty_GdprCookie`
+
+### Test magepack locally
+
+To test magepack locally:
+* clear and enable cache,
+* enable mergin, minify and magepack budnling in your db:
```
Comment (JKrupinski) [inline README.md:528]: Typo `mergin`

File: README.md [hunk 3]
```diff
@@ -481,3 +481,73 @@ If responisve image aspect ratio is added, additional styles inline are generate
...
+bin/magento config:set dev/js/minify_files 1
+bin/magento config:set dev/css/minify_files 1
+bin/magento config:set dev/css/merge_css_files 1
+```
+* run tools compilation for production:
+```
+yarn styles --prod && yarn babel --prod && yarn svg
+```
+* generate magepack config:
+yarn magepachGenerate ..
```
Comment (JKrupinski) [inline README.md:541]: magepach? :D

File: README.md (hunk 1 above)
Comment (anqaka) [reply README.md:509]: 👍

File: README.md (hunk 2 above)
Comment (anqaka) [reply README.md:528]: 👍

File: README.md (hunk 3 above)
Comment (anqaka) [reply README.md:541]: :D master of typo, fixed :D thx

**Pull request merged**